### PR TITLE
introduces attributes object to paper

### LINKED ISSUE #294: Introduce object containing arguments
Currently certain, less important, arguments are directly being passed into Paper.
It would be clearer to use a named object instead.

### PR DESCRIPTION
**Pull request by terw-dan**
Creates an attributes object with defaults for keyword and description. 

Fixes #294

### CODE REVIEW

File: js/values/Paper.js [hunk 1]
```diff
@@ -1,22 +1,14 @@
...
-
-	// if there is no metaData object given, use an empty object so checks on metaData keep working
-	if ( isUndefined( metaData ) ) {
-		metaData = {};
-	}
-
-	this._metaDescription = metaData.metaDescription || "";
+	this._attributes = attributes || {};
+	this._keyword = this._attributes.keyword || "";
+	this._metaDescription = this._attributes.metaDescription || "";
```
Comment (jcomack) [inline js/values/Paper.js]: This 'check' should probably be handled somewhere else in the code, seeing as it's not a hard requirement.

**Comment by jcomack** [comment]
CR done :+1:

File: js/values/Paper.js [hunk 2]
```diff
@@ -1,22 +1,14 @@
...
+var Paper = function( text, attributes ) {
 	this._text = text || "";
-
-	// if there is no metaData object given, use an empty object so checks on metaData keep working
-	if ( isUndefined( metaData ) ) {
-		metaData = {};
-	}
-
-	this._metaDescription = metaData.metaDescription || "";
+	this._attributes = attributes || {};
```
Comment (atimmer) [inline js/values/Paper.js]: We could also handle this with `defaultsDeep`

Comment (terw-dan) [inline js/values/Paper.js]: I've used `defaults` now, since we only have a single level, so `defaultsdeep` isn't necessary.

**Comment by jcomack** [comment]
CR 2 done :+1:

**Comment by jcomack** [comment]
Acceptance done. Will merge once builds are completed.

**Pull request merged**